test/functional/p2p_addrv2_relay.py

```diff
@@ -89,8 +96,8 @@ def run_test(self):
         msg.addrs = ADDRS
         msg_size = calc_addrv2_msg_size(ADDRS)
         with self.nodes[0].assert_debug_log([
-                f'received: addrv2 ({msg_size} bytes) peer=0',
-                f'sending addrv2 ({msg_size} bytes) peer=1',
+                f'received: addrv2 ({msg_size} bytes) peer=1',
+                f'sending addrv2 ({msg_size} bytes) peer=2',
```

Comment:
(I think this is a good example of why we shouldn't use `assert_debug_log` when we're able to check the logic otherwise)

Reply:
Yes :)